pipelines/smartseq2_by_plate/SmartSeq2Plate.wdl

```diff
@@ -0,0 +1,112 @@
+import "SmartSeq2SingleSample.wdl" as single_cell_run
+import "SmartSeq2PlateAggregation.wdl" as ss2_plate_aggregation
+       
+workflow RunSmartSeq2ByPlate {
+  meta {
+    description: "The RunSmartSeq2ByPlate pipeline runs multiple SS2 samples in a single pipeline invocation"
+  }
+
+  # Version of this pipeline
+  String version = "RunSmartSeq2ByPlate_v0.0.1"
```

Comment:
Maybe “MulticellSmartSeq2_v###”? or “multisample”? This could run on more or less than a plate, right? Im unclear if typically our workflow names match our version names, but seems simpler if it does. It’d be good for the workflow to also say multi-X instead of plate.

What would it take to make it a v1.0?